docs/source/docker/security/security.rst

```diff
@@ -0,0 +1,83 @@
+Security
+==========
+
+Security and data privacy is very important to us. In this section
+you find all security related docs. 
+Legal documents such as Privay Notice, T&C, DPA are available under
+`lightly.ai/legal <https://lightly.ai/legal>`_.
+
+.. note:: These docs apply to all personal, team and custom account plans.
+
+Architecture Overview
+----------------------
+
+Here you find an overview of the cloud architecture. 
+
+**A few important things to note:**
+
+- Data storage and processing happens on clients infrastructure 
+- Lightly only needs permission to list the cloud bucket and to create signed URLs
+- Any additional data besides the actual images such as metadata, predictions or
+  any other non-sensitive data used to manage the datasets is stored in secured
+  databases within the Lightly Cloud
+- Authentication is provided through our partner Auth0. Additional services such 
+  as 2FA, SAML can be added upon request
+
+.. figure:: images/lightly-cloud-architecture.png
+    :align: center
+    :alt: Image of Lightly Architecture
+    :figclass: align-center
+
+How is the data flowing around?
+-------------------------------
+
+We differentiate between usage data and the actual samples. Samples will be mostly 
```

Comment:
Samples will be mostly images or videos -> Samples can be images or videos and their subtypes such as sequences, frames or object crops.